test/razorIntegrationTests/formatting.integration.test.ts

```diff
@@ -32,8 +32,8 @@ jestLib.describe(`Razor Formatting ${testAssetWorkspace.description}`, function
         const htmlConfig = vscode.workspace.getConfiguration('html');
         await htmlConfig.update('format.enable', true);
 
-        await integrationHelpers.activateCSharpExtension();
         await integrationHelpers.openFileInWorkspaceAsync(path.join('Pages', 'BadlyFormatted.razor'));
+        await integrationHelpers.activateCSharpExtension();
```

Comment:
interesting... not sure why this would help.  Potentially some timing thing where the file needs to be opened for a bit to trigger something on the Razor side?  Weird